lib/config.js

```diff
@@ -8,5 +8,6 @@ module.exports = {
     "sync_frequency" : 5,
     "storage_strategy": "dom",
     "do_console_log": false
-  }
+  },
+  topicPrefix: 'wfm:cloud:'
```

Comment:
This should be `cloudTopicPrefix` as it is referencing the `wfm:cloud` topic name.